src/zenml/utils/pipeline_docker_image_builder.py

```diff
@@ -265,10 +266,21 @@ def build_docker_image(
             )
             requirements_file_names = [f[0] for f in requirement_files]
 
+            apt_packages = docker_settings.apt_packages
+            if docker_settings.install_stack_requirements:
+                apt_packages += stack.apt_packages
```

Comment:
would this still work if we have the same package in both `docker_settings.apt_packages ` and `stack.apt_packages`?

Reply:
Yep that works without any issues from what I've tested